src/main/java/com/parsing/pdf/parsing/SaverLotPDFResult.java

```diff
@@ -0,0 +1,29 @@
+package com.parsing.pdf.parsing;
+
+import com.parsing.model.LaptopItem;
+import com.parsing.repository.LaptopItemRepository;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+import java.math.BigDecimal;
+
+@Component
+@RequiredArgsConstructor
+@Slf4j
+public class SaverLotPDFResult {
```

Comment:
LotPDFResultSaver? maybe we can delegate it to LotPDFResultService? if we have such a model?

Reply:
Done.